dev/deps/spark-deps-hadoop-3.2

```diff
@@ -211,6 +211,7 @@ spire-macros_2.12-0.17.0-M1.jar
 spire-platform_2.12-0.17.0-M1.jar
 spire-util_2.12-0.17.0-M1.jar
 spire_2.12-0.17.0-M1.jar
+spotbugs-annotations-3.1.9.jar
```

Comment:
Uh oh, this is LGPL. We can't include this. https://spotbugs.github.io/
It looks like ZK specifically excludes it from their binary release: https://github.com/apache/zookeeper/commit/372e713a9d2d9264417313e5d68e9437ffddd0f5
Therefore, we probably just need to write an exclusion rule to not pull in this dependency, as it sounds like it's not strictly needed at runtime (just annotations)

Reply:
@srowen Thank you for your very detailed investigation. I learned more. I will try to exclude it.